rest-api-spec/src/main/resources/rest-api-spec/api/security.update_api_key.json

```diff
@@ -0,0 +1,38 @@
+{
+  "security.update_api_key": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-update-api-key.html",
```

Comment:
Need to write the docs still 😅

Reply:
Merging the REST spec  without a doc page will cause downstream doc build to fail. We should add stub docs with this PR. An example is https://github.com/elastic/elasticsearch/pull/76670

Reply:
Thanks for the pointer! Will do.